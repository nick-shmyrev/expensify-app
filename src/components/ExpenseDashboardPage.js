import React from 'react';

import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpensesSummary from './ExpensesSummary';

const ExpenseDashboardPage = () => (
  <div>
    <h1>Expense Dashboard</h1>
    <ExpenseListFilters/>
    <ExpensesSummary/>
    <ExpenseList/>
  </div>
);

export default ExpenseDashboardPage;
