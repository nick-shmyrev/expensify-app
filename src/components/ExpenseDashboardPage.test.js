import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from './ExpenseDashboardPage';

describe('ExpenseDashboardPage component', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<ExpenseDashboardPage/>);
    expect(wrapper).toMatchSnapshot();
  });
});
