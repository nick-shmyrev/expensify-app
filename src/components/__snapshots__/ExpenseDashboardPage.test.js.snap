// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpenseDashboardPage component should render correctly 1`] = `
<div>
  <h1>
    Expense Dashboard
  </h1>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
